### DISCUSSION TITLE
No module named 'pymysql'

### DISCUSSION THREAD

**Discussion by ProdigyMaster** [sqlalchemy/sqlalchemy#7852] [Usage Questions]
### Describe the bug

Hi 👋
When connecting to the database I have got an error No module named 'pymysql' . But this module is clearly installed:

![image](https://user-images.githubusercontent.com/528501/159885158-3564ff12-12b0-4a94-b8c9-920cc7e6a295.png)

Thanks for any help :-) 

Windows 10, Anaconda

### To Reproduce

```python
from sqlalchemy import create_engine

engine = create_engine('mysql+pymysql://<db_user>:<db_pass>@<db_host>:<db_port>/<db_name>')
connection = engine.connect()
query = "Select name FROM company WHERE company_id = 83326"
ResultProxy = connection.execute(query)
ResultSet = ResultProxy.fetchall()
print(ResultSet)
```

### Error

ModuleNotFoundError: No module named 'pymysql'

### Versions

- OS: Windows
- Python: 3.8.12
- SQLAlchemy: Latest
- Database: mysql

**Comment by CaselIT** [comment]
Hi,

This seems like a problem with your environment, not something where sqlalchemy has control over.
Does running `import pymysql` work in the when placed in a file? If not then try to reinstall it or to recreate the environment
